tests/test_dag_tree/test_dag_tree.py

```diff
@@ -1,34 +1,29 @@
 """Tests for the dag_tree module."""
 
-from __future__ import annotations
```

Comment:
I removed this for testing the input type dicts, since otherwise type annotations will be available only as strings.

Reply:
Cool. But won't this be the behaviour in the future and we are shooting ourselves in our future selves' feet this way?